app/register/page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test';
+
+const goodUser = {
+  email: 'test1@email.com',
+  password: 'test12345',
+  confirmPassword: 'test12345',
+};
+
+const badUser = {
```

Comment:
nit: idk if it's necessary to break this into two different variables... what are your thoughts? I might rewrite this to have it all in the `user` variable

```tsx
import { test, expect } from '@playwright/test';

const user = {
  confirmPassword: 'test12345',
  email: 'test1@email.com',
  incorrectEmail: 'test@email.com',
  incorrectPassword: 'test',
  password: 'test12345',
};

test.beforeEach(async ({ page }) => {
  await page.goto('/register');
});

test.describe('Tests register page', () => {
  test('should successfully register', async ({ page }) => {
    await page.getByTestId('email').fill(user.email);
    await page.getByTestId('password').fill(user.password);
    await page.getByTestId('confirm-password').fill(user.confirmPassword);
    await page.getByTestId('continue-button').click();
    await page.goto('/weeklyPicks');
    await expect(page).toHaveURL('/weeklyPicks');
  });
  test('should not be able to register and register button should be disabled', async ({
    page,
  }) => {
    await page.getByTestId('email').fill(user.email);
    await page.getByTestId('password').fill(user.password);
    await page.getByTestId('confirm-password').fill(user.incorrectPassword);
    const isDisabled = await page.getByTestId('continue-button').isDisabled();
    expect(isDisabled).toBe(true);
    await expect(page).toHaveURL('/register');
  });
});
```

Reply:
nit: @ryandotfurrer: I was going to suggest to use the same credentials (ie objects) from the `Login` page just for consistency sake (of course with the extra key-value for `confirmPassword`).

Reply:
> nit: @ryandotfurrer: I was going to suggest to use the same credentials (ie objects) from the `Login` page just for consistency sake (of course with the extra key-value for `confirmPassword`).

@CorinaMurg Thank you! I meant to mention that as well 😓

Reply:
@ryandotfurrer and @CorinaMurg I did keep same user's email information for badUser. 

As for 'goodUser' on this test, I had to create a different user because the current user from 'Login' was already created and logged in, therefore, would result in an error. I definitely could have updated 'Login's correctCredentials' with 'Register's goodUser' but since this test is in regards to 'Register', I did not want to fiddle with 'Login' code. 

I hope this makes sense!

Reply:
Using the same email and passwords for both Login and Registration should not matter. The tests are independent from each other, and Playwright creates a separate/new browser context for each.

What I had in mind was the entire object, including the name of the object (`correctCredentials`, `incorrectCredentials`). Makes it easier to understand that in each case (ie page) these objects play a similar role. Again, this is just a nit-level comment.

Reply:
You can create an object that housed both separate objects. Then, it's just 1 object.

Reply:
@CorinaMurg the email and password that are currently used for 'Login' right now is already stored in the database, therefore, I won't be able to use that to register a new user, hence a different email and password are used. If I use the same password and email that Login has, the test will fail since the user has already been captured. I hope this makes sense :). Please let me know if it doesn't and I can clarify.